### PR TITLE
[W5.11][W15-A4] Tan Kai Yong - enhancement 1

### PR DESCRIPTION
**Pull request by Kyholmes**
modified find command to search keywords with case-insensitive against person names.
convert both keywords and person names to lower case before splitting and compare.

### CODE REVIEW

File: src/seedu/addressbook/commands/FindCommand.java [hunk 1]
```diff
@@ -17,14 +13,15 @@
     public static final String COMMAND_WORD = "find";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all persons whose names contain any of "
-            + "the specified keywords (case-sensitive) and displays them as a list with index numbers.\n"
+            + "the specified keywords (case-insensitive) and displays them as a list with index numbers.\n"
```
Comment (amrut-prabhu) [inline src/seedu/addressbook/commands/FindCommand.java:16]: Good work for remembering to update the messages

**Comment by Yoochard** [comment]
Nice work for turning into case-insensitive!

File: src/seedu/addressbook/commands/FindCommand.java [hunk 2]
```diff
@@ -1,10 +1,6 @@
 package seedu.addressbook.commands;
 
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Set;
+import java.util.*;
```
Comment (eldriclim) [inline src/seedu/addressbook/commands/FindCommand.java:3]: Avoid importing by `*`
You may leave it as it is for now, however in 4kloc, such coding standards will be enforced, you might want to refer to the coding standards for the module.

File: src/seedu/addressbook/commands/FindCommand.java [hunk 3]
```diff
@@ -57,4 +54,19 @@ public CommandResult execute() {
...
     }
 
+    /**
+     * convert the keywords set to string array, iterate the array to convert the string to lower case and convert back to set again
+     */
+    private void changeKeywordToLowerCaseInSet()
+    {
+        String[] keywordArray = this.keywords.toArray(new String[keywords.size()]);
+
+        for(int i = 0; i< keywordArray.length; i++)
```
Comment (eldriclim) [inline src/seedu/addressbook/commands/FindCommand.java:64]: Likewise for keywords like `if` `while` `for`, please refer to the coding standards for the module.

**Review by eldriclim — COMMENTED** [review]
Good job on PR. Please close PR after reading the comments.

**Pull request closed without merging**